docs/pages/api/routes.md

```diff
@@ -43,64 +43,14 @@ type UpdateScene = {
 ```typescript
 type RegisterInputStream = {
   type: "register";
-  entity_type: "rtp_input_stream";
-  input_id: string;
-  port: Port;
-  video?: Video;
-  audio?: Audio;
+  entity_type: "rtp_input_stream" | "mp4";
+  ... // input specific options
 }
 ```
+See inputs documentation to learn more.
 
-Parameters of registered RTP input stream. Before using input in video composition or output mixing, input has to be firstly registered using `register_input` request.
-
-At least one of `video` and `audio` has to be defined.
```

Comment:
make sure that the generated descriptions are on par with what was here. It does not have to be the same, but check if everything makes sense. It's possible that some fixed landed here in the past, but code comments were not updated